### DISCUSSION TITLE
ACE 3.5 for Arma reforger?

### DISCUSSION THREAD

**Discussion by xzTWINzx** [acemod/ACE3#9811] [General]
Will your team develop an ace mod for arma reforger? Do you have any plans? Or will you skip this part?

**Comment by jonpas** [comment]
> Due to severe limitations of the Enfusion engine and its scripting language, the lack of an in-game editor and most importantly an active playerbase we currently aren't working on bringing ACE to Arma Reforger.

If you want to discuss modding on Reforger, we do have channels setup on our Discord that you are welcome to join.
